src/Microsoft.DotNet.Darc/src/Darc/Operations/SetRepositoryMergePoliciesOperation.cs

```diff
@@ -124,6 +124,13 @@ public override async Task<int> ExecuteAsync()
                 mergePolicies = initEditorPopUp.MergePolicies;
             }
 
+            IRemote verifyRemote = RemoteFactory.GetRemote(_options, repository, Logger);
```

Comment:
NIT: This pattern of GetRemote && VerifyMaestroManagedBranchExists appeared a few times.. Perhaps it would deserve a refactoring.